user/e16/template.py

```diff
@@ -0,0 +1,61 @@
+pkgname = "e16"
+pkgver = "1.0.30"
+pkgrel = 0
+build_style = "gnu_configure"
+configure_args = [
+    "--prefix=/usr",
+    "--sysconfdir=/etc",
+    "--mandir=/usr/share/man",
+    "--localstatedir=/var",
+    "--enable-sound",
+    "--enable-sound-alsa",
```

Comment:
we generally don't enable alsa in things when they also support pulse/pipewire etc, unless 'alsa' means something like 'uses alsa for midi' (but not for basic audio output)

Reply:
OK that's really helpful context to have for packaging this, and for future packages. I've removed ALSA support as it's an alternate backend e16 can use to speak to ALSA devices directly. With the pulse/libpulse support sound will work just fine with ALSA disabled as long as pulseaudio or pipewire is running, so it's not an issue to disable ALSA if the preference is for pulse support to be used.